src/common/components/app/app.scss

```diff
@@ -48,3 +48,17 @@ footer {
     margin-bottom: var(--spacing-s) !important;
   }
 }
+
+.haitaton-pagination {
+  margin-top: var(--spacing-s);
+
+  span[class*='item-link'] {
+    width: var(--link-height);
+  }
+}
+
+.haitaton-button-icon-size-initial {
```

Comment:
Thinking about the same here if we could get rid of the extra divs in so many places.

Reply:
Removed extra divs but left this global class as I thought it's more convenient to apply it to few buttons that need it than to create wrapper component.